dfsclient/src/test/java/org/hpccsystems/dfs/client/DFSReadWriteTest.java

```diff
@@ -67,6 +67,8 @@ public void readBadlyDistributedFileTest() throws Exception
         //this file only has data on two nodes
         HPCCFile file = new HPCCFile(datasets[1], connString , hpccUser, hpccPass);
         file.setProjectList("");
+        file.setFileAccessExpirySecs(10);
```

Comment:
These values are always tough to predict, why not make it configurable?
Also, is 10 secs meant to ensure we don't time out? If so, how close do we usually come to expiring before finishing?

Reply:
I changed the timeout to reduce the time it takes for the tests to run. At the moment in readFile() there is code that purposely causes a timeout to test if reading can continue on active connections after the timeout is up. That is one of the reasons the full suite of tests was taking a while to run. I reduced the timeout from 120s -> 10s to speed up the tests.

Reply:
Removing those changes for now. Will address in a separate PR